js/src/run_trees.ts

```diff
@@ -289,16 +306,17 @@ export class RunTree implements BaseRun {
   }
 
   private static getDefaultConfig(): object {
+    const start_time = Date.now();
     return {
-      id: uuid.v4(),
+      id: uuidFromTime(start_time),
```

Comment:
This is misleading, let's refactor this a bit to use `_serialized_start_time` as I mentioned above

Or just remove `id` from this altogether